### PR TITLE
feat: WindowNotificationManager: support TopCenter and BottomCenter position

### PR DESCRIPTION
**Pull request by rabbitism**
## What does the pull request do?

Add TopCenter and BottomCenter option to WindowNotificationManager. 
These are frequently used. 

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0039240-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

File: src/Avalonia.Themes.Fluent/Controls/WindowNotificationManager.xaml [hunk 1]
```diff
@@ -42,6 +42,11 @@
       <Setter Property="HorizontalAlignment" Value="Right" />
     </Style>
 
+    <Style Selector="^:topright /template/ ReversibleStackPanel#PART_Items">
```
Comment (jp2masa) [inline src/Avalonia.Themes.Fluent/Controls/WindowNotificationManager.xaml:45]: Should be `topcenter`?

Comment (rabbitism) [reply src/Avalonia.Themes.Fluent/Controls/WindowNotificationManager.xaml]: Thanks. fixed.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0039257-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by rabbitism** [comment]
Hi consider backport this to 11.0.x?

**Comment by maxkatz6** [comment]
I can add a tack, but no more backports are planned for 11.0.x, unless there are critical bugs or customer's request.

**Pull request merged**